independent-projects/tools/common/src/test/resources/templates/README.ftl

```diff
@@ -0,0 +1,34 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
```

Comment:
I would prefer not having 2 versions of this file. Dunno why you need that one?

Reply:
I just find all the other `.ftl` files in two places. I did the same thing for this one.
I think one is for testing the templating and the other for building.

If I'm wrong, could you explain me why `.ftl` files are duplicated ?

Reply:
done